packages/js/manifests/polywrap/src/formats/polywrap/migrate.ts

```diff
@@ -10,30 +10,15 @@ import {
   PolywrapManifestFormats,
   latestPolywrapManifestFormat
 } from ".";
-
-import {
-  migrate as migrate_0_1_0_to_0_2_0
-} from "./migrators/0.1.0_to_0.2.0";
-
-type Migrator = {
-  [key in PolywrapManifestFormats]?: (m: AnyPolywrapManifest) => PolywrapManifest;
-};
-
-export const migrators: Migrator = {
-  "0.1.0": migrate_0_1_0_to_0_2_0,
-};
+import { findShortestMigrationPath } from "../../migrations";
+import { migrators } from "./migrators";
 
 export function migratePolywrapManifest(
   manifest: AnyPolywrapManifest,
   to: PolywrapManifestFormats
 ): PolywrapManifest {
   let from = manifest.format as PolywrapManifestFormats;
 
-  // HACK: Patch fix for backwards compatability
-  if(from === "0.1" && ("0.1.0" in migrators)) {
```

Comment:
You can take a look at e.g. `packages/js/manifests/polywrap/src/formats/polywrap.app/migrators/index.ts` where you will find a migration to `0.2.0` both from `0.1` and `0.1.0`. It's still a slight hack, but it feels less hacky to me, as we are explicitly defining this migration instead of relying on a patch fix inside the `migrate...` function.

Reply:
Sounds good!